### ISSUE TITLE
4_MRAI_case_study.ipynb is not using only 6 cases for testing

### ISSUE THREAD

**Issue by Warvito** [estherpuyol/MRAI_workshop#1]
Hi Esther!

There is a little bug in the code of 4_MRAI_case_study.ipynb.
In order to train the model to perform the segmentation, the dataset is divided between training and test set. However, it is not used 6 samples for testing as mentioned. The code is using the caseIndices[-6,0] value (= 368) to divide the dataset, resulting in a training set with 368 samples and a test set with 52 samples.
